Cargo.toml

```diff
@@ -13,4 +14,4 @@ default-members = ["ogcapi"]
 license = "MIT OR Apache-2.0"
 repository = "https://github.com/georust/ogcapi"
 edition = "2021"
-rust-version = "1.66"
```

Comment:
Is Rust 1.75 required? I've got requests from users that crates should be compilable with the Rust version of Debian Stable, which is 1.63...

Reply:
I'm fine with omitting `rust-version`. Optimally, the minimal version required is determined by checking the rustc changelog or with a tool like https://github.com/foresterre/cargo-msrv

The latter is quite time consuming, since it builds the code with different toolchain versions. For me, the Debian toolchain is not an important requirement, just a reminder to be conservative using brand new compiler features.

Reply:
I am somewhat biased on this as I think we should not try to accomodate some subjective level of conservativity. My policy was always what is in current stable is fine. If users wants to be conservative they have to take care of it themselves and maybe wait out on new releases until it works for their environment. Then again, this is just me pushing away responsibility and keep my life as simple as possible ;)